modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSourceReact.testcase

```diff
@@ -65,9 +65,29 @@ definition {
 	@ignore = "Test Stub"
 	@priority = 4
 	test CanNotPublishWithoutTitle {
```

Comment:
Yeah, I saw this one when I was automating the test. I wasn't sure where else I should change the test's name because I know that you guys are keeping track of the automations in spreadsheets, and because of that, I didn't change its name.